Cartfile

```diff
@@ -15,4 +15,4 @@ github "SwiftyJSON/SwiftyJSON"              "3.1.4"
 github "farhanpatel/JSONSchema.swift"       "master"
 github "google/EarlGrey"                    "1.11.0"
 github "jhugman/SwiftRouter"                "master"
-github "mozilla-mobile/telemetry-ios"       "master"
+github "mozilla-mobile/telemetry-ios"       "v1.0.6"
```

Comment:
did you verify if this is the proper syntax? (i.e. starting with the "v")

Reply:
Yeah the exact tag name is the one to use.